README.md

```diff
@@ -27,23 +36,32 @@ ________________________________________________________________________________
 
 Once you boot, give it a couple of minutes and the Fedora setup screen will appear, just as it would after a desktop install. Create your username, password, etc. I'd also recommend changing the root password to something other than "linux". After you're set up, open a terminal and type:
 
-# sudo passwd root
+```bash
+sudo passwd root
+```
 
 Then change it.
 
 After that, you'll need to resize the root partition to fill the entirety of your disk. So type:
  
+```bash
 # sudo cfdisk /dev/mmcblkX
```

Comment:
i think you forgot to remove "#" from that line

Reply:
Thanks for the review, fixed in faed8e3993d6e0300eebfe6a2819a62127c0f541!